src/ng/compile.js

```diff
@@ -212,10 +212,22 @@
  *
  *
  * #### `bindToController`
- * When an isolate scope is used for a component (see above), and `controllerAs` is used, `bindToController: true` will
+ * This property can be used to bind scope properties directly to the controller. It can be either
+ * `true` or an object hash with the same format as the `scope` property. Additionally,
+ * `controllerAs` must be used for it to work.
+ *
+ * When an isolate scope is used for a directive (see above), `bindToController: true` will
```

Comment:
But how can you bind to the controller if there's nothing to bind to (i.e. if you have scope: true, then there are no bindings, right)?